doggos/deployments/doggos.yaml

```diff
@@ -1,21 +1,21 @@
 apiVersion: apps/v1
 kind: Deployment
 metadata:
-  name: OWNER-doggos
+  name: varowner-doggos
   labels:
     app: doggos
-    owner: OWNER
+    varowner: varowner
```

Comment:
shouldn't the label name stay as `owner`?

Reply:
oh whoops, good call.